src/web/views/Components/Menu/props-basic-menu.js

```diff
@@ -33,4 +33,10 @@ export default [
     "true",
     "If false, the menu component does not close when you click outside.",
   ],
+  [
+    "childrenLeft",
+    "Boolean",
+    "false",
+    "If true, the child menu will open to the left of the menu.",
```

Comment:
Present instead of future ?

Reply:
"the child menu opens to the left of the menu"

Reply:
Already present, but not added to the documentation